### PR TITLE
chore: Remove beta/preview environment separation (RHINENG-10786)

### PR DESCRIPTION
**Pull request by leSamo**
https://issues.redhat.com/browse/RHINENG-10786

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: Test
```
Comment (mkholjuraev) [inline .github/workflows/test.yml:1]: I believe we have test runs with Travis already. Do we need to have another test run in GitHub workflows? Is not this an extra?

File: config/overrideChrome.js [hunk 2]
```diff
@@ -1,5 +1,4 @@
 export default () => ({
   updateDocumentTitle: () => undefined,
-  isBeta: () => true,
```
Comment (mkholjuraev) [inline config/overrideChrome.js:3]: isBeta is not going away according to the platform. Should we leave as is this for now

**Review by mkholjuraev — APPROVED** [review]
LGTM!

File: .github/workflows/test.yml (hunk 1 above)
Comment (leSamo) [reply .github/workflows/test.yml:1]: True, I don't remember why is this stage duplicated

File: config/overrideChrome.js (hunk 2 above)
Comment (leSamo) [reply config/overrideChrome.js:3]: It was not used in the code so it did not need to be overridden, but I'll keep it for future use.

Comment (leSamo) [reply config/overrideChrome.js:3]: Hm, actually, [`isBeta` is deprecated](https://github.com/RedHatInsights/insights-chrome/blob/8205b3043192f297f05e0c94444bd75f1ca0f722/src/utils/VisibilitySingleton.ts#L90) and should not be used anymore, so it does make sense to remove it

**Pull request merged**